### PR TITLE
updates for proxies and planning

### PR DESCRIPTION
**Pull request by herbeck558**
added annotations for proxies needed; added GODEBUG env for oauth proxy to disable http 2 (caused http hang up errors under load). planning capacity update

### CODE REVIEW

File: src/pages/security/manage-access.md [hunk 1]
```diff
@@ -9,8 +9,21 @@ toc: true
...
+If you have applications that run outside of the cluster and want to provide secure access to the ACD service in the cluster there are several options to consider to limit access to the ACD service: 
+ 1 At a network layer, the firewall / network security group (nsg) can restrict incoming access to the openshift route used to externalize the ACD service to a limited set of networks or systems.   
+ 2 Use a client certificate with mutual TLS - see [openshift documentation](https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html#nw-mutual-tls-auth_configuring-ingress). 
+ 3 Provide access to ACD through a reverse proxy that layers on security.  One way to do this is with the OpenShift-provided OAuth service with a [proxy](https://github.com/openshift/oauth-proxy) and a service account to do role-based access control (RBAC) access to the service.  The rest of this page will document how to set this proxy up for protecting an ACD service. Note, with this option, no direct route to ACD is exposed from the cluster and all traffic flows thorugh the proxy which adds some overhead and another hop to traffic.
```
Comment (cagrev) [inline src/pages/security/manage-access.md:15]: typo on "thorugh", should be "through"

File: src/pages/security/manage-access.md [hunk 2]
```diff
@@ -80,6 +100,9 @@ In the example below, we'll use a 4.8 version of the OpenShift OAuth proxy.  See
             - name: oauth-proxy
               image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.8
               imagePullPolicy: IfNotPresent
+              env:
+              - name: GODEBUG
```
Comment (cagrev) [inline src/pages/security/manage-access.md:104]: Wonder if we want to call out a doc reference for this variable? Maybe this, https://pkg.go.dev/runtime#hdr-Environment_Variables, or was there some other reading that you found helpful? Consumers may question setting a debug variable without knowing/understanding what it is used for. I'll let you decide if it's necessary or not.

**Review by cagrev — APPROVED** [review]
Just a minor comment on a typo.

File: src/pages/security/manage-access.md [hunk 3]
```diff
@@ -9,8 +9,21 @@ toc: true
...
-In the example below, we'll use a 4.8 version of the OpenShift OAuth proxy.  See [instructions here](https://catalog.redhat.com/software/containers/openshift4/ose-oauth-proxy/5cdb2133bed8bd5717d5ae64?container-tabs=gti) for how to pull this image.  You can update the version to match your OpenShift version.  We set the replica count to 2 which gives some level of high-availability (HA) and distributes the workload some.  You may need to increase the replica count based on load.
+If you have applications that run outside of the cluster and want to provide secure access to the ACD service in the cluster there are several options to consider to limit access to the ACD service: 
+ 1 At a network layer, the firewall / network security group (nsg) can restrict incoming access to the openshift route used to externalize the ACD service to a limited set of networks or systems.   
+ 2 Use a client certificate with mutual TLS - see [openshift documentation](https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html#nw-mutual-tls-auth_configuring-ingress).
```
Comment (dlangst) [inline src/pages/security/manage-access.md:14]: Same comment here - "OpenShift" versus ""openshift"

File: src/pages/security/manage-access.md [hunk 4]
```diff
@@ -9,8 +9,21 @@ toc: true
...
+
+In the first 2 options above you'll create a route direct to the ACD service and rely on the network security to secure access.  In this case we recommend these route annotations:
+```
+   annotations:
+          haproxy.router.openshift.io/balance: leastconn
+          haproxy.router.openshift.io/disable_cookies: 'true'
+          haproxy.router.openshift.io/timeout: 90s
+```
+
+For the 3rd option follow the steps below to setup a Oauth proxy in a different namespace and expose a route to that.
```
Comment (dlangst) [inline src/pages/security/manage-access.md:25]: For consistency, "OAuth" versus "Oauth"

**Review by dlangst — APPROVED** [review]

**Pull request merged**